src/samples/EducationStart/reuseables/PegaSetup.tsx

```diff
@@ -471,6 +472,13 @@ export const useStartMashup = (
     }; */
   }, []);
 
+  useEffect(() => {
+    if (sdkIsLoggedIn()) {
+      CheckAuthAndRedirectIfTens();
+      setShowResolutionPage(true);
+    }
+  }, [showPega]);
```

Comment:
As discussed, we are manipulating showPega state with respect to different scenarios. Also, we don't need resolution page during Tens check. 
This can be removed and Tens check can be handled by checking 'pCoreReady + sdkIslogedIn' state in index file.